tests/unit/Codeception/Module/TestsForWeb.php

```diff
@@ -1292,4 +1292,12 @@ public function testFillFieldWithAmpersand()
         $form = data::get('form');
         $this->assertEquals('this & that', $form['name']);
     }
+
+    public function testSwitchToIframe()
+    {
+        $this->module->amOnPage('/iframe');
+        $this->module->switchToIframe('content');
+        $this->module->see('Is that interesting?');
+        $this->module->click('Ссылочка');
```

Comment:
I got a very misleading exception: [MalformedLocatorException] XPath or CSS locator is malformed: Sign In! 
I can see why it failed now.